### PR TITLE
Update notes.md

### PR DESCRIPTION
**Pull request by stereoplegic**
Added alternative to persist Clover screen resolution

### CODE REVIEW

File: notes.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ return an integer value. For this particular case, select the nearer 1360x768
 resolution instead. VNC implementation may require a multiple of 16, so beware
 of setting 1080 vertical resolution or use SPICE instead.
 
+Alternative Clover screen resolution note by stereoplegic: Clover Configurator resulution not sticking for me, but editing the desired Clover screen resolution in `HighSierra/config.plist.stripped.qemu` or `Mojave/config.plist.stripped.qemu` and following instructions in the {README.md for High Sierra](HighSierra/README.md) or [README.md for Mojave](Mojave/README.md) cause the set resolution to persist in Clover (EDK II OVMF UEFI instructions still apply, especially to avoid issues in fullscreen caused by resolution change in Clover).
```
Comment (kholia) [inline notes.md]: Too many references here. Can we simplify this?

Comment (stereoplegic) [reply notes.md]: By that, do you mean I should just go with Mojave and omit both High Sierra references, or more?

Comment (kholia) [reply notes.md]: Thanks. Documented at Thanks. Documented at https://github.com/kholia/OSX-KVM/blob/master/notes.md#higher-resolution-uefi--clover now.

Comment (kholia) [reply notes.md]: I have added you to the CREDITS file.

**Pull request closed without merging**